runner/worker.go

```diff
@@ -83,6 +85,12 @@ func (w *Worker) makeRequest(tv TickValue) error {
 
 	ctd := newCallData(w.mtd, w.workerID, reqNum, !w.config.disableTemplateFuncs, !w.config.disableTemplateData, w.config.funcs)
 
+	if w.mtd.IsClientStreaming() || w.mtd.IsServerStreaming() {
```

Comment:
Are those true for bidi streaming? (I assume both will be true, but want to confirm)

Reply:
Yes both will be true. The same pattern is used for bidi calls further down on line 167